tests/cases/compiler/lateBoundClassMemberAssignmentJS.ts

```diff
@@ -0,0 +1,17 @@
+// @allowJs: true
+// @checkJs: true
+// @noEmit: true
+// @strict: true
+// @target: es6
+// @filename: file.js

```

Comment:
it would be nicer to put this in conformance/salsa with the rest of the assignment declaration tests

also it would be nice to use a filename that matches the on-disk filename (except extension).